pages/users/[id].tsx

```diff
@@ -177,51 +201,131 @@ export default function User({ loginContext }: Props) {
       <Navbar
         loginContext={loginContext}
         fill="black"
-        className="bg-ifpink text-black"
+        className={clsx('bg-ifpink', 'text-black')}
       />
 
-      <main className="bg-ifpink flex-1 justify-center flex pt-16 pb-32">
+      <main
+        className={clsx(
+          'bg-ifpink',
+          'flex-1',
+          'justify-center',
+          'flex',
+          'pt-16',
+          'pb-32',
+          'w-full',
+          'overflow-hidden'
+        )}
+      >
         <div style={{ flexBasis: 1138 }}>
           <OffsetBorderContainer>
-            <div className="px-24 pt-16 pb-12">
+            <div
+              className={clsx(
+                'px-5',
+                'md:px-8',
+                'tablet:px-24',
+                'pt-5',
+                'md:pt-8',
+                'tablet:pt-16',
+                'pb-12'
+              )}
+            >
               {/* Header */}
               <div
-                className="flex justify-between mb-8"
+                className={clsx('flex', 'justify-between', 'md:mb-8')}
                 style={{ width: '100%' }}
               >
-                <div>
-                  <h1 className="font-extended text-6xl mt-6 mb-4">
+                <div className={clsx('flex', 'flex-col')}>
+                  <h1
+                    className={clsx(
+                      'font-extended',
+                      'text-3xl',
+                      'md:text-6xl',
+                      'mt-2',
+                      'md:mt-6',
+                      'mb-4',
+                      'max-w-[14rem]',
+                      'md:max-w-[48rem]',
+                      'max-h-[4rem]',
+                      'overflow-hidden'
```

Comment:
I think this can clip graffitis in the tablet view -- I'm not sure what the best UX is here, but maybe just an ellipsis on overflow would clear up that it's truncated?
<img width="805" alt="image" src="https://user-images.githubusercontent.com/767083/156237883-7383a9a5-3625-4084-874d-4930a0835cc5.png">

Reply:
For this one I increased the possible width and also added an ellipsis.